src/plugin-slots/LoginPage/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { PluginSlot } from '@openedx/frontend-plugin-framework';
+import PropTypes from 'prop-types';
+
+import LoginPage from '../../login/LoginPage';
+
+const LoginPageSlot = ({
+  institutionLogin,
+  handleInstitutionLogin,
+}) => (
```

Comment:
I don't know how valid this is. The original component needs these values, however they are not included in plugin props since they seem pretty specific to the implementation of the login page.

Reply:
Basing on many similar code examples [here](https://github.com/search?q=org%3Aopenedx%20PluginSlot&type=code), I think it's valid, @xitij2000?